store/mockstore/unistore/cophandler/mpp.go

```diff
@@ -327,46 +338,6 @@ type MPPTaskHandler struct {
 	Err    error
 }
 
-// HandleMPPDispatch handle DispatchTaskRequest
```

Comment:
Is this function not called previously?

Reply:
It did be called. For now I move it to the unistore and polish it.